sdc/datatypes/hpat_pandas_series_rolling_functions.py

```diff
@@ -864,56 +915,88 @@ def _impl(self, other=None, pairwise=None, ddof=1):
         minp = self._min_periods
 
         main_series = self._data
+        main_arr = main_series._data
```

Comment:
Can'd you reuse function used for corr?

Reply:
There are a few differences in the implementations of `cov` and `cor`: parameter `ddof` and alignment by finiteness of input data (replacement of infinite values with nans). So I separated the implementations as it was before.